src/components/shared/FilterNavigatorBar.css

```diff
@@ -38,6 +38,14 @@
 
 .filterNavigatorBarItemContent {
   overflow: hidden;
+
+  /* The 8px are used by the box-shadow when the button receives focus */
+  height: calc(100% - 8px);
+  padding: 0;
+  margin: auto;
+  border: none;
+  background: none;
+  color: inherit;
```

Comment:
We need a few more lines to make the font look right on my computer, otherwise it's too big:
 ```css
font: inherit;
line-height: normal;
```
On my computer, once we add these, then we don't need the `height` trick above. I believe the `line-height` does the trick. Then can you remove it as well (along with the comment)?